.github/workflows/cherryPick.yml

```diff
@@ -100,20 +100,21 @@ jobs:
           USER: OSBotify
           TITLE_REGEX: Update version to ${{ env.NEW_VERSION }}
 
-      - name: Cherry-pick the merge commits to new branch
-        id: cherryPick
+      - name: Cherry-pick the version-bump to new branch
         run: |
           git fetch
-          git cherry-pick ${{ steps.getCPMergeCommit.outputs.MERGE_COMMIT_SHA }} ${{ steps.getVersionBumpMergeCommit.outputs.MERGE_COMMIT_SHA }} --mainline 1
-        continue-on-error: true
+          git cherry-pick -x --mainline 1 --strategy=recursive -Xtheirs ${{ steps.getVersionBumpMergeCommit.outputs.MERGE_COMMIT_SHA }}
 
-      # If there is a merge conflict, we'll just commit what we have,
-      # and the PR will be auto-assigned for someone to manually resolve the conflicts.
-      - name: If there is a merge conflict...
-        if: ${{ steps.cherryPick.outcome == 'failure' }}
+      - name: Cherry-pick the merge commit of target PR to new branch
+        id: cherryPick
         run: |
-          git add -A
-          git cherry-pick --continue
+          if ./.github/scripts/cherryPick.sh ${{ steps.getCPMergeCommit.outputs.MERGE_COMMIT_SHA }}; then
+            echo "CP was a success, PR can be automerged 🎉"
+            echo "::set-output name=SHOULD_AUTOMERGE::true"
```

Comment:
Can this be in the bash script too?

Reply:
I just wonder if this entire step could just be `./.github/scripts/cherryPick.sh` ?

Reply:
I think we _could_, but I was sort of thinking that having GH-actions specific stuff like step outputs "hidden" in the script was a bit 😬

Sort of similar to your critique of having [this line](https://github.com/Expensify/Expensify.cash/blob/ed2f64bb20abc9ae37faeffc9ffca0bf53a62e57/.github/libs/GithubUtils.js#L32) "hiding" in the GithubUtils (something I've been meaning to clean up)

Reply:
That's fair - Maybe I will pose the opposite suggestion, I do wonder if the script is simple enough now where it could just live in the step? The first `if` will be one line, the second being three lines

Reply:
Yeah, I can make that change! I really only made the script separate to take advantage of ShellCheck in the IDE (because I suck at bash scripting). I'll nix the script and put this all in the workflow